ipi/engine/forcefields.py

```diff
@@ -1275,3 +1275,490 @@ def poll(self):
                     r["t_finished"] = time.time()
                     self.gather(r)
                     r["status"] = "Done"
+
+
+class PhotonDriver:
+
+    """
+    Photon driver for a single cavity mode
+    """
+
+    def __init__(
+        self, apply_photon=True, E0=1e-4, omega_c_cminv=3400.0, ph_rep="loose"
+    ):
+        """
+        Initialise PhotonDriver
+
+        In this implementation, the photonic masses are set as 1 a.u.
+
+        Args:
+            apply_photon: Determine if applying light-matter interactions
+            E0: varepsilon in the paper, light-matter coupling strength
+            omega_c_cminv: cavity frequency in units of cm-1
+            ph_rep: loose / dense: if the photon coordinates are stored as 2*Nmodes or Nmodes photon "atoms"
```

Comment:
I am not sure I understand this description, could you clarify it?

Reply:
Just a ping to @taochem, because this point seems to be overlooked

Reply:
I would suggest adding ( please check the help string in ipi/inputs/for... for a more verbose explanation )